virttest/utils_conn.py

```diff
@@ -964,11 +992,13 @@ def conn_setup(self, server_setup=True, client_setup=True):
             self.server_setup()
         if client_setup:
             self.client_setup()
+        if server_setup_local:
+            self.server_setup(on_local=True)
```

Comment:
after this step, there are two servers will be setup, is this expected?

Reply:
yes. This is as expected.